CMakeLists.txt

```diff
@@ -69,8 +69,23 @@ if (TARGET_PLATFORM STREQUAL "openwrt")
         add_definitions(-DBEEROCKS_UGW)
         set(BEEROCKS_NOTIFY 1)
         set(BEEROCKS_REPEATER_MODE 1)
-    elseif (TARGET_PLATFORM_TYPE STREQUAL "turris-omnia")
-        set(BEEROCKS_WLAN2_IFACE "wlan1")
+    else()
+        # For generic openwrt target allow to override default interface names
+        if (NOT DEFINED ENV{PRPLMESH_WLAN1_IFACE})
+            set(BEEROCKS_WLAN1_IFACE "wlan0")
+        else()
+            set(BEEROCKS_WLAN1_IFACE "$ENV{PRPLMESH_WLAN1_IFACE}")
+        endif()
+        if (NOT DEFINED ENV{PRPLMESH_WLAN2_IFACE})
+            set(BEEROCKS_WLAN2_IFACE "wlan1")
+        else()
+            set(BEEROCKS_WLAN2_IFACE "$ENV{PRPLMESH_WLAN2_IFACE}")
+        endif()
+        if (NOT DEFINED ENV{PRPLMESH_BH_WIRE_IFACE})
+            set(BEEROCKS_BH_WIRE_IFACE "eth1")
+        else()
+            set(BEEROCKS_BH_WIRE_IFACE "$ENV{PRPLMESH_BH_WIRE_IFACE}")
+        endif()
```

Comment:
Since all of this will anyway be deprecated by #574 I think we should go for the simple solution:

```
... ugw stuff ...
else()
    set(BEEROCKS_WLAN2_IFACE "wlan1")
endif()
```

Reply:
Do you mean revert it to what it was, just set BEEROCKS_WLAN2_IFACE to "wlan1"?
At minimum I need to be able to set wired backhaul to eth0 fro Gl.Inet and  keep it eth1 fro Turris. I don't mind removing all the wlan# interface stuff if it will no longer be needed soon.